### DISCUSSION TITLE
Why can't I see logs for in-progress runs?

### DISCUSSION THREAD

**Discussion by allanlewis** [renovatebot/renovate#27994] [Request Help] [answered]
### What would you like help with?

Other

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

In the "recent jobs" list, I can only see logs for completed jobs. Is there a reason I can't see logs for "running" jobs? Could this be added?

### Logs (if relevant)

n/a

**Comment by rarkins** [comment, marked as answer]
It's not technically feasible without a lot of work, so not planned

**Reply by allanlewis** [reply]
OK, understood. Perhaps the error message could be updated to explain that logs for in-progress runs aren't possible, rather than:

> No log is available for this job
> Something went wrong and the log is not available

**Reply by rarkins** [reply]
Please create a "Suggest an idea" discussion for specifically this

**Reply by allanlewis** [reply]
Sure: https://github.com/renovatebot/renovate/discussions/28599
